packages/polymorphic/src/Polymorphic/index.ts

```diff
@@ -9,15 +8,3 @@ export type {
   PolymorphicPropsWithRef,
   PolymorphicRef,
 } from './Polymorphic.types';
-
-expect.extend({
-  toBePolymorphic,
-});
-
-declare global {
-  namespace jest {
-    interface Matchers<R> {
-      toBePolymorphic(): R;
-    }
-  }
-}
```

Comment:
This causes issues when importing the component. We'll need to extend expect in testing lib https://jira.mongodb.org/browse/LG-2733

Reply:
This needs to be merged in before my PR that converts typography off of box and onto polymorphic